### DISCUSSION TITLE
Having trouble compiling

### DISCUSSION THREAD

**Discussion by Fury1012** [KadeArchive/Kade-Engine#2522] [Q&A]
I can't compile without it saying this, anyone know how to fix it? 
![pain](https://user-images.githubusercontent.com/45504212/137595398-9502d057-7b8d-4a7e-810d-ae5d159f8e2e.PNG)

**Comment by teotm** [comment]
Can you show me Character.hx code?

**Comment by Jotaro-Gaming** [comment]
> I can't compile without it saying this, anyone know how to fix it? ![pain](https://user-images.githubusercontent.com/45504212/137595398-9502d057-7b8d-4a7e-810d-ae5d159f8e2e.PNG)

Replace the : with ;
